kolibri/plugins/device/assets/src/modules/pluginModule.js

```diff
@@ -9,6 +9,7 @@ export default {
   state() {
     return {
       authenticateWithPin: false,
+      isDevicePlugin: false,
```

Comment:
This vuex module is only used in the device plugin, so this will always be true when this is being used. Better to find another way to handle this conditionality, wherever you might be using it.